app/renderer/store/workbench/sagas.js

```diff
@@ -359,10 +359,20 @@ export function* createNewRealFile({ payload }){
         return; // Save As dialog was canceled by user
     }
     
-    // C:\projects\cb-webui\WebAPI\aaz.js => C:\projects\cb-webui\WebAPI\
-    let folderPath = saveAsPath.split("\\");
-    folderPath.pop();
-    folderPath = folderPath.join("\\");
+    let folderPath;
+
+    if (process.platform === 'win32') {
+        // C:\projects\cb-webui\WebAPI\aaz.js => C:\projects\cb-webui\WebAPI\
+        folderPath = saveAsPath.split("\\");
```

Comment:
Could have used [path.sep](https://nodejs.org/api/path.html#path_path_sep) instead of doing separate checks for windows/posix